pom.xml

```diff
@@ -85,47 +79,69 @@
         <commons-pool.version>1.6</commons-pool.version>
-        <protobuf.version>3.1.0</protobuf.version>
-        <servicemix.bundles.junit.version>4.11_2</servicemix.bundles.junit.version>
+        <gson.version>2.7</gson.version>
+        <javax.servlet-api.version>3.1.0</javax.servlet-api.version>
+        <guava.version>18.0</guava.version>
+        <abdera.version>1.1.3</abdera.version>
+        <abdera.osgi.version>1.1.3_2</abdera.osgi.version>
+        <bouncy.version>1.55</bouncy.version>
+        <dropwizard.version>3.1.2</dropwizard.version>
+        <httpclient.version>4.5.3</httpclient.version>
+        <httpcore.version>4.4.6</httpcore.version>
+        <jaxb.version>2.2.11</jaxb.version>
+        <joda-time.version>2.9.4</joda-time.version>
+        <germonimo-servlet-api.group>org.apache.geronimo.specs</germonimo-servlet-api.group>
+        <germonimo-servlet-api.artifact>geronimo-servlet_3.0_spec</germonimo-servlet-api.artifact>
+        <geronimo-servlet-api.version>1.0</geronimo-servlet-api.version>
+        <woodstox.core.version>4.4.1</woodstox.core.version>
+        <woodstox.stax2-api.version>3.1.4</woodstox.stax2-api.version>
+        <wss4j.version>2.1.9</wss4j.version>
+        <validation.version>1.1.0.Final</validation.version>
+        <xpp3.bundle.version>1.1.4c_6</xpp3.bundle.version>
+        <jboss-logging.version>3.3.0.Final</jboss-logging.version>
+        <javax-el.version>3.0.0</javax-el.version>
+        <xmlsec.version>2.0.8</xmlsec.version>
+        <dom4j.bundle.version>1.6.1_5</dom4j.bundle.version>
+        <jdom.bundle.version>1.1_4</jdom.bundle.version>
+        <tika.version>1.14</tika.version>
+        <!--END versions used in Camel and CXF features in platform-app-->
+
+        <!--If these bundles are used elsewhere in DDF, the versions MUST exactly match the versions used by the CXF and Camel features.-->
+        <saxon.version>9.6.0-4</saxon.version> <!--org.apache.servicemix.bundles/org.apache.servicemix.bundles.saxon/-->
+        <saxon.bundle.plugin>${saxon.version}_1</saxon.bundle.plugin>
+        <xpp3.bundle.version>1.1.4c_6</xpp3.bundle.version> <!--org.apache.servicemix.bundles/org.apache.servicemix.bundles.xpp3/-->
+        <dom4j.bundle.version>1.6.1_5</dom4j.bundle.version> <!--org.apache.servicemix.bundles/org.apache.servicemix.bundles.dom4j/-->
+        <jdom.bundle.version>1.1_4</jdom.bundle.version> <!--org.apache.servicemix.bundles/org.apache.servicemix.bundles.jdom/-->
+        <jsr339-api.bundle.version>2.6.0</jsr339-api.bundle.version> <!--org.apache.servicemix.specs/org.apache.servicemix.specs.jsr339-api-2.0.1/-->
+        <woodstox.stax2-api.version>3.1.4</woodstox.stax2-api.version> <!--org.codehaus.woodstox/stax2-api/-->
+        <woodstox.core.version>4.4.1</woodstox.core.version> <!--org.codehaus.woodstox/woodstox-core-asl/-->
+
```

Comment:
@vinamartin, maybe we should add a comment at the end of this section of properties, like `<!--END of versions MUST exactly match the versions used by the CXF and Camel features.-->`, so that properties won't be accidentally appended to this list